### DISCUSSION TITLE
Howto for using zigbee2mqtt in Homeassistant Core

### DISCUSSION THREAD

**Discussion by Arno-v** [Koenkk/zigbee2mqtt#14674] [General]
I am trying to get zigbee2mqtt with homeassistant running in a FreeBSD 13.1 jail. I have installed Homeassistant and Mosquitto, anf both are running fine. I adda Mosquitto as MQTT broker in Homeassistant, and, as far as I can tell, this also runs fine.

I have build Zigbee2Mqtt, and this also starts, and seems to function.
Now Homeassistant core does not have an add-on store, so it is not possible to use the automatic way to add the Zigbee2mqtt addon to Homeassistant. Is there anyway to handle this manually by editting some yaml file(s)?

**Comment by kalbfuss** [comment]
Hi Arno-V,
you do not need a z2m home assistant add-on. Integration is done via the MQTT integration. Means, if mosquitto is up and running and both, home assistant and z2m connect properly, then you should start to see zigbee devices from z2m in home assistant.
